doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -17,10 +17,37 @@
 <a name="4.1"></a>
 ## 4.1 Introduction
 
+This chapter will describe in detail the Kubernetes Reference Architecture in terms of the functional capabilities and how they relate to the Reference Model requirements, i.e. how the infrastructure profiles are delivered and described.
+
+Figure 4-1 below shows the architectural components that are described in the subsequent sections of this chapter.
+
+<p align="center"><img src="../figures/ch04_k8s_architecture.png" alt="Kubernetes Reference Architecture" Title="Kubernetes Reference Architecture" width="65%"/></p>
+<p align="center"><b>Figure 4-1:</b> Kubernetes Reference Architecture</p>
+
 <a name="4.2"></a>
 ## 4.2 Host OS
 
-> This chapter should describe the requirements for a host os to run the software stack of Reference Architecture 2.
+In order for a Host OS to be compliant with this Reference Architecture it must meet the following requirements:
+- A deb/rpm compatible distribution of Linux (this must be used for the master nodes, and can be used for worker nodes).
+- A version of Linux that is [compatible with kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#before-you-begin) - this has been chosen as the baseline because kubeadm is focussed on installing and managing the lifecycle of Kubernetes and nothing else, hence it is easily integrated into higher-level and more complete tooling for the full lifecycle management of the infrastructure, cluster add-ons, etc.
+- Windows Server 2019 (this can be used for worker nodes, but be aware of the [limitations](https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#limitations)).
+- In order to support `req.gen.cnt.03` and `req.lcm.gen.01` and `req.lcm.adp.04`, the Host OS must be disposable and therefore the configuration of the Host OS (and associated infrastructure such as VM or bare metal server) must be consistent - e.g. the system software and configuration of that software must be identical apart from those areas of configuration that must be different such as IP addresses and hostnames.
+
+Table 4-1 lists the Operating Systems that comply with this Reference Architecture specification.
+
+|OS|Version(s)|Notes|
+|---|---|---|
+|Ubuntu|16.04+||
+|Debian|9+||
+|CentOS|7||
```

Comment:
CentOS / RHEL 8?

Reply:
At present, RHEL8 isn't compatible with kubeadm.  As we progress we can add more based on other LCM tools, but I've started with just kubeadm as the "native" tool from the Kubernetes project.  i.e. I'd rather get this published as-is and add more post-Snezka.  Is that ok?

Reply:
well, K8s also doesn't officially support e.g. multi-tenancy, Docker versions after 17. something etc., but still we state requirements :)
but I understand your reasoning, no strong feelings one way or the other
however if we state CentOS/RHEL 7 is okay, then we need discuss kernel versions too IMO